lib/project_types/script/layers/domain/extension_point.rb

```diff
@@ -62,6 +62,16 @@ def versioned?
           end
         end
       end
+
+      class UnknownExtensionPoint < ExtensionPoint
+        def initialize(type)
+          super(type, {
+            "libraries" => {
+              "typescript" => {}
+            }
```

Comment:
This system only supports one language at the moment. The new Script API creation guide will be using typescript, so this should be sufficient.